From 8f24f04ad31717753eaaef9da4590c157d382c69 Mon Sep 17 00:00:00 2001
From: Petros Angelatos <petrosagg@gmail.com>
Date: Thu, 7 Sep 2017 11:52:15 +0300
Subject: [PATCH 1/2] Revert "runtime: check and fail early with a message if
 MMX is not available on 386"

This reverts commit 35e84546d7c3cc599dd5bf31ad86bf8d3d841d04.

Signed-off-by: Petros Angelatos <petrosagg@gmail.com>
---
 src/runtime/asm_386.s | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/runtime/asm_386.s b/src/runtime/asm_386.s
index 4181859724..a6ef392402 100644
--- a/src/runtime/asm_386.s
+++ b/src/runtime/asm_386.s
@@ -75,9 +75,6 @@ notintel:
 	MOVL	AX, runtime·cpuid_ecx(SB)
 	MOVL	DX, runtime·cpuid_edx(SB)
 
-	// Check for MMX support
-	TESTL	$(1<<23), DX	// MMX
-	JZ 	bad_proc
 
 nocpuinfo:	
 
-- 
2.14.1

